cmd/evalhub_mcp/main_test.go

```
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestVersionFlag(t *testing.T) {
+	cmd := exec.Command("go", "run", ".", "--version")
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)